geode-core/src/main/java/org/apache/geode/internal/cache/ClusterConfigurationLoader.java

```diff
@@ -280,19 +281,9 @@ public ConfigurationResponse requestConfigurationFromLocators(String groupList,
 
     ConfigurationResponse response = null;
     for (InternalDistributedMember locator : locatorList) {
```

Comment:
would it be faster if one locator didn't respond, we move on to the next locator to try instead of trying 6 times on one unresponsive locator first? since each retry would wait 10 seconds in between. Of course this wouldn't make any difference if we only have one locator, but if we have multiple locators, we should iterate through all the locators first before we try 6 times on all locators.

Reply:
I'll take another look, but I don't think it's possible to distinguish between the situation when the locator is completely gone vs. the cluster config service is not available yet.

Reply:
you don't need to. Just put the retry logic outside of the iterating over locators.

Reply:
I think I've got it cleaner now. I've targeted some specific Exceptions too.